src/reports/models/reportsManager.js

```diff
@@ -35,9 +35,11 @@ module.exports.getReports = async (testId, filter) => {
 
     const reportSummaries = await databaseConnector.getReports(testId, filter, contextId);
     const config = await configHandler.getConfig();
-    const reports = reportSummaries.map((summaryRow) => {
-        return getReportResponse(summaryRow, config);
-    });
+    const reports = await Promise.all(reportSummaries.map(async (summaryRow) => {
+        const experiments = await getChaosExperimentsByJobId(summaryRow.job_id);
```

Comment:
when you use promise all you must use try catch. best to do it in the internal function (line 38-9), so if one fails it still ok.